de.metas.swat/de.metas.swat.base/src/main/sql/postgresql/system/40-sw01_swat/5452470_sys_FRESH-783_QualityNote_Table.sql

```diff
@@ -0,0 +1,239 @@
+-- 26.10.2016 17:56
+-- URL zum Konzept
+INSERT INTO AD_Table (AccessLevel,ACTriggerLength,AD_Client_ID,AD_Org_ID,AD_Table_ID,CopyColumnsFromTable,Created,CreatedBy,EntityType,ImportTable,IsActive,IsAutocomplete,IsChangeLog,IsDeleteable,IsHighVolume,IsSecurityEnabled,IsView,LoadSeq,Name,ReplicationType,TableName,Updated,UpdatedBy) VALUES ('3',0,0,0,540794,'N',TO_TIMESTAMP('2016-10-26 17:56:12','YYYY-MM-DD HH24:MI:SS'),100,'de.metas.swat','N','Y','N','N','Y','N','N','N',0,'Qualität-Notiz','L','M_QualityNote',TO_TIMESTAMP('2016-10-26 17:56:12','YYYY-MM-DD HH24:MI:SS'),100)
+;
+
+-- 26.10.2016 17:56
+-- URL zum Konzept
+INSERT INTO AD_Table_Trl (AD_Language,AD_Table_ID, Name, IsTranslated,AD_Client_ID,AD_Org_ID,Created,Createdby,Updated,UpdatedBy) SELECT l.AD_Language,t.AD_Table_ID, t.Name, 'N',t.AD_Client_ID,t.AD_Org_ID,t.Created,t.Createdby,t.Updated,t.UpdatedBy FROM AD_Language l, AD_Table t WHERE l.IsActive='Y' AND l.IsSystemLanguage='Y' AND l.IsBaseLanguage='N' AND t.AD_Table_ID=540794 AND NOT EXISTS (SELECT * FROM AD_Table_Trl tt WHERE tt.AD_Language=l.AD_Language AND tt.AD_Table_ID=t.AD_Table_ID)
+;
+
+-- 26.10.2016 17:56
+-- URL zum Konzept
+CREATE SEQUENCE M_QUALITYNOTE_SEQ INCREMENT 1 MINVALUE 0 MAXVALUE 2147483647 START 1000000
```

Comment:
Please separate DDL and DML and add a commit, between the two, to avoid "pending trigger events" error